packages/fast-components-react-msft/src/card/README.md

```diff
@@ -0,0 +1,10 @@
+# Card
+*Cards* are snapshots of content that are typically used in a group to present collections of related information. *Cards* can represent any quickly consumable information.  The *card* component can be either a [div](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/div), [article](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article), or [section](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) element.
+
+## Usage
+Use *cards* when displaying a feed of content or when a stronger isolation between each content items is needed. *Cards* are also well suited for scenarios where the content is not consistent and requires different fields, image sizes, or content, which is often the case in a feed. *Cards* create an opportunity to present a collection of related content in a rich and dynamic way. *Cards* offer flexible layout options that can be used interchangeably with cards using other options; often a requirement when the content available is not consistent.
```

Comment:
"*Cards* offer flexible layout options... " Is this true in this PR? I don't see anything with that functionality. I would say we should take it out if we aren't offering it.